e2eRunner/src/main/scala/za/co/absa/hermes/e2eRunner/TestDefinitionJsonProtocol.scala

```diff
@@ -17,6 +17,13 @@ package za.co.absa.hermes.e2eRunner
 
 import spray.json.{DefaultJsonProtocol, RootJsonFormat}
 
+import scala.util.matching.Regex
+
 object TestDefinitionJsonProtocol extends DefaultJsonProtocol {
   implicit val pluginDefinitionFormat: RootJsonFormat[TestDefinition] = jsonFormat6(TestDefinition.apply)
+
+  val VarsField = "vars"
+  val RunsField: String = "runs"
+  val VarsPattern: Regex = """(?<=#\{)\w*(?=\}#)""".r
```

Comment:
Consider adding a couple of unit tests to test the pattern itself. It would serve as documentation as well. Unit tests would provide info on which patterns should match, and which shouldn't.

Reply:
Add also some comments to these constants.